brave/brave-core
# Fixes locals being split for uphold

Resolves https://github.com/brave/brave-browser/issues/5477

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- go to rewards page
- make sure that Uphold in wallet panel is bold

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
